packages/integration-tests/src/fixtures/hosts/index.ts

```diff
@@ -29,17 +27,15 @@ const getUpdatedHosts = () => {
     return [];
   }
 
-  const lastSignal = Cypress.env('AI_LAST_SIGNAL');
+  const lastSignal: string = Cypress.env('AI_LAST_SIGNAL');
```

Comment:
Is  `const lastSignal: string` always a string? What if `AI_LAST_SIGNAL` is undefined?

Reply:
Since we asserted at the beginning that we are in an scenario where at least 1 host is discovered, it would always have a value.